sched/environ/env_dup.c

```diff
@@ -78,7 +78,7 @@ int env_dup(FAR struct task_group_s *group, FAR char * const *envcp)
 
   /* Is there an environment ? */
 
-  if (envcp || (envcp = ptcb->group->tg_envp))
+  if (envcp || (envcp = ptcb->group->tg_envp) != NULL)
```

Comment:
Wow... What's a finding. I'm wondering how the things were working till now. Let me double check if it shouldn't be

```
  if (envcp || (envcp == ptcb->group->tg_envp) != NULL)
```

Reply:
Ok. Already addressed